### PR TITLE
#184638373 user login

### PR DESCRIPTION
**Pull request by oreste-abizera**
#### What does this PR do?
 
 implements user login

#### Description of  the task to be completed?

A user can log into the Barefoot Nomad application via email & password

#### How should this be manually tested?

- checkout into `ft-user-login-184638373` branch and install dependencies including jsonwebtoken
- add `JWT_SECRET` and `JWT_EXPIRES_IN` in the env file
- register a user and then send a POST request to ${Basic URL}/api/users/login and provide email and password in the body

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#184638373](https://www.pivotaltracker.com/story/show/184638373)

#### Screenshots (if appropriate)
...

### CODE REVIEW

**Review by felixdusengimana — COMMENTED** [review]
LGTM!

**Review by felixdusengimana — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -42,7 +43,8 @@
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.29.3",
     "swagger-jsdoc": "^6.2.8",
-    "swagger-ui-express": "^4.6.2"
+    "swagger-ui-express": "^4.6.2",
+    "swaggiffy": "^1.0.5"
```
Comment (felixdusengimana) [inline package.json]: remove this package "swaggiffy"

**Review by felixdusengimana — CHANGES_REQUESTED** [review]

**Review by dushimsam — APPROVED** [review]
Looks great

File: package.json (hunk 1 above)
Comment (oreste-abizera) [reply package.json]: removed

**Review by dianepretty — APPROVED** [review]

**Review by edine-noella — APPROVED** [review]

**Review by felixdusengimana — APPROVED** [review]
LGTM

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]
Please remove unwanted files and folders like swaggify file and swagger folder which is out of src folder

**Review by honorine22 — APPROVED** [review]

**Comment by oreste-abizera** [comment]
> Please remove unwanted files and folders like swaggify file and swagger folder which is out of src folder

Done

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**